metricbeat/module/ceph/osd_df/_meta/docs.asciidoc

```diff
@@ -0,0 +1,3 @@
+=== Ceph cluster_health metricset
```

Comment:
Also this title can now be removed with our new script. Sorry missed that previously but realised it now that i checked out the branch locally.